src/Http/Http.Results/src/UnprocessableEntity.cs

```diff
@@ -1,32 +1,29 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace Microsoft.AspNetCore.Http;
-
+using Microsoft.AspNetCore.Http.Metadata;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
+namespace Microsoft.AspNetCore.Http.HttpResults;
+
 /// <summary>
 /// An <see cref="IResult"/> that on execution will write an object to the response
 /// with Unprocessable Entity (422) status code.
 /// </summary>
-public sealed class UnprocessableEntityObjectHttpResult : IResult
+public sealed class UnprocessableEntity : IResult, IEndpointMetadataProvider
```

Comment:
@DamianEdwards Is it too late to rename these to `Unprocessable Content` to match the new official reason phrase from [RFC9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-422-unprocessable-content)?

Reply:
Also filed https://github.com/dotnet/aspnetcore/issues/42302